.evergreen/evergreen.yml.in

```diff
@@ -1084,6 +1086,7 @@ buildvariants:
     tasks:
       - name: e2e_tests_linux_x64
       - name: e2e_tests_linux_x64_openssl11
+      - name: e2e_tests_linux_x64_openssl11_fips
```

Comment:
This is the only variant where FIPS support worked out of the box. For other OSes, I guess the docker tests have to be enough (although I was pleasantly surprised to see that FIPS support works even for their docker images).